pages/mysubmissions.js

```diff
@@ -0,0 +1,15 @@
+import Layout from '../components/Layout'
+import MySubmissions from '../components/MySubmissions'
+import { Typography } from '@material-ui/core'
+
+const MySubmissionsPage = () => {
+  return (
+    <Layout>
+      <Typography variant="h2">My Submitted Links</Typography>
+      <br/>
```

Comment:
Maybe add the form or a link to add new content here as well at the end of the submissions list?

Reply:
Good idea, done!